service/project_membership.go

```diff
@@ -38,8 +38,10 @@ func (s *ProjectMembershipService) CreateInvitation(
 		return model.ProjectInvitation{}, err
 	}
 
-	if _, err := s.projectSvc.Get(ctx, c.ProjectID, authUserID); err != nil {
+	if exists, err := s.projectSvc.Exists(ctx, c.ProjectID, authUserID); err != nil {
 		return model.ProjectInvitation{}, err
+	} else if !exists {
+		return model.ProjectInvitation{}, apierrors.NewProjectNotFoundError()
 	}
```

Comment:
Clearer error handling

Reply:
Yes 👍 
But still, not 100% smooth 😄 
See comment below 😉